backend/runner/plancheck/statement_report_executor.go

```diff
@@ -127,7 +128,16 @@ func (e *StatementReportExecutor) runForDatabaseTarget(ctx context.Context, conf
 		sqlDB := driver.GetDB()
 
 		return reportForPostgres(ctx, sqlDB, database.DatabaseName, renderedStatement, dbSchema.GetMetadata())
-	case storepb.Engine_MYSQL, storepb.Engine_OCEANBASE:
+	case storepb.Engine_OCEANBASE:
+		driver, err := e.dbFactory.GetAdminDatabaseDriver(ctx, instance, database)
+		if err != nil {
+			return nil, err
+		}
+		defer driver.Close(ctx)
+		sqlDB := driver.GetDB()
+
+		return reportForOceanBase(ctx, sqlDB, instance.Engine, database.DatabaseName, renderedStatement, dbSchema.GetMetadata())
+	case storepb.Engine_MYSQL:
```

Comment:
Why treat MySQL and OceanBase differently here?

Reply:
because OceanBase use its own getAffectedRows. https://github.com/bytebase/bytebase/pull/9553/files#diff-7b3ad62eb8c4fbc6cadb0144a01139ba8aff07f535dee4c37aba0c8284c4db65R236